user/inc/Arduino.h

```diff
@@ -193,6 +202,11 @@ typedef volatile uint32_t RwReg;
 // C++ only
 #ifdef __cplusplus
 
+// In order to support compiling with Clang, explicitly include cmath here since the Clang version
+// of the math.h header included above does not include cmath. This is a no-op compiling with GCC
+// since the GCC version of math.h includes cmath. Replacing the math.h include above
+#include <cmath>
```

Comment:
This is now included twice.

Reply:
Gah, sorry, I caught this locally but forgot to push. Fixed now.